tests/test_metadata_models.py

```diff
@@ -85,11 +85,9 @@ def _test_opencontext_material_model(opencontext_source_path):
             "mat:anyanthropogenicmeterial",
             "mat:biogenicnonorganicmaterial",
             "mat:mineral",
+            "mat:organicmaterial",
```

Comment:
This needs to be fixed again in soon future, when we update the OpenContext model by training it on the new Getty AAT mappings.